lib/linguist/heuristics.yml

```
@@ -840,3 +846,20 @@ named_patterns:
   raku: '^\s*(?:use\s+v6\b|\bmodule\b|\b(?:my\s+)?class\b)'
   vb-class: '^\s*VERSION\s+[0-9]\.[0-9]\s+CLASS'
   vb-form: '^\s*VERSION\s+[0-9]\.[0-9]{2}'
+  vb-module: '^\s*Attribute\s+VB_Name\s+=\s+'
```

Redundant qualifier that I missed earlier before:

```suggestion
  vb-module: '^\s*Attribute\s+VB_Name\s+=\s'
```